Xamarin.Forms.Platform.Android/VisualElementPackager.cs

```diff
@@ -67,44 +70,49 @@ protected virtual void Dispose(bool disposing)
 
 		public void Load()
 		{
-			SetElement(null, _renderer.Element);
+			SetElement(null, _element);
 		}
 
 		void AddChild(VisualElement view, IVisualElementRenderer oldRenderer = null, RendererPool pool = null, bool sameChildren = false)
 		{
 			Performance.Start();
 
-			if (_childViews == null)
-				_childViews = new List<IVisualElementRenderer>();
+			if (CompressedLayout.GetIsHeadless(view)) {
+				var _packager = new VisualElementPackager(_renderer, view);
```

Comment:
the underscore in the variable name is a little confusing here, I think.

Reply:
fixed. at some point that variable was a field...